web/history.md

```diff
@@ -1,11 +1,7 @@
 # KeymanWeb Version History
 
-## 2019-01-03 11.0.201 beta
-* New Feature:
-  * Adds the alignInputs() API function to facilitate touch-alias element work-arounds in case of future issues. (#69)
-  
-* Bug Fix:
-  * Fixed keyboard layer transition bug on mobile devices. (#978)
+## 2019-01-04 11.0.202 beta
+* Fixed external keyboard keys "tab" and "backspace" for embedded platforms (#1474)
```

Comment:
Why are we removing these recent lines from history?

Reply:
ack, I missed it in my rebase